docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md

```diff
@@ -0,0 +1,54 @@
+# Installing Intel Data Center Attestation Primitives (DCAP) and its dependencies on Windows
+
+
+## [Intel Platform Software for Windows (PSW) v2.4](http://registrationcenter-download.intel.com/akdlm/irc_nas/15654/Intel%20SGX%20PSW%20for%20Windows%20v2.4.100.51291.exe)
+
+After unpacking the self-extracting ZIP executable, install the *PSW_EXE_RS2_and_before* version:
+```cmd
+C:\Intel SGX PSW for Windows v2.3.100.49777\PSW_EXE_RS2_and_before\Intel(R)_SGX_Windows_x64_PSW_2.3.100.49777.exe
```

Comment:
This should probably have the 2.4 directory instead of the 2.3

Reply:
@johnkord I am following the latest docs from here: https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/GettingStarted.Windows.md
And with Sergio's latest changes, this is the 2.3 executable. @jazzybluesea  please comment here

Reply:
Fixed it.